scala/lambdas/dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormatters.scala

```diff
@@ -260,4 +262,5 @@ object DynamoFormatters {
 
     case PreservationRepresentationType, AccessRepresentationType
 
+  case class Checksum(algorithm: String, fingerprint: String)
```

Comment:
There is no particular reason other than it is better than "value". The word value is so overloaded that it loses its meaning on its own. I was tossing up between "fingerprint" and "hash", then decided to avoid "hash"  because hash felt like it describes the algorithm and "the value" together (i.e. what we are calling checksum) Hence I homed in on "fingerprint". I am not fussed about it but I would also argue if we want to change it to value, why not call the "checksum" as "fixity" as well

Reply:
Hmm, not sure. @kurtismash @MancunianSam , what do you think?

Reply:
Wikipedia: 
`In computer science, a fingerprinting algorithm is a procedure that maps an arbitrarily large data item (such as a computer file) to a much shorter bit string, its fingerprint, that uniquely identifies the original data for all practical purposes`

Reply:
I think `case class Hash(algorithm: String, checksum: String)` is probably going a bit far :smile: There's a lot of things called checksum already and I don't want to change those.

It would be nice not to have something else called value along with the many other things we've already got so I vote for `case class Checksum(algorithm: String, fingerprint: String)`